contexts/ed25519-signature-2020-v1.json

```diff
@@ -0,0 +1,89 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "@protected": true,
+
+    "Ed25519VerificationKey2020": {
+      "@id": "https://w3id.org/security#Ed25519VerificationKey2020",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "controller": {
+          "@id": "https://w3id.org/security#controller",
+          "@type": "@id"
+        },
+        "privateKeyMultibase": {
+          "@id": "https://w3id.org/security#privateKeyMultibase",
+          "@type": "https://w3id.org/security#multibase"
+        },
```

Comment:
```suggestion
```

This feels like a footgun -- we don't want to easily enable people to publish public key information in LD documents. If we want to define this, define it in another context file and name it DANGER_CONTEXT.

Reply:
Removed.